docs/source/configuration/spans.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Spans
+subtitle: Add router lifecycle context to traces
+description: Use spans to add contextual information from the Apollo Router to traces displayed by your application performance monitors (APM).
+---
+
+import RouterServices from '../../shared/router-lifecycle-services.mdx';
+
+A **span** captures contextual information about requests and responses as they're processed through the [router's request lifecycle (pipeline)](../customizations/overview/#the-request-lifecycle). The information from spans can be used when displaying traces in your application performance monitors (APM).
```

Comment:
It seems like we use "lifecycle" in some places and "pipeline" in others. Can we select one term and use it consistently?

Reply:
Agree, not sure which one we want to use though.

Reply:
True, though tt looks as though "request lifecycle" is a little more ubiquitous in the router docs. (E.g. https://www.apollographql.com/docs/router/customizations/overview/#the-request-lifecycle).

That said, I lean slightly towards "pipeline" in the router context, since we're focusing on what's happening in the router rather than the entire lifecycle of a request from when a client sends it until it receives a response.

Before making any  changes, would love @shorgi 's take once he's back.